plugins/src/webrtcsink/imp.rs

```diff
@@ -371,6 +372,16 @@ fn make_converter_for_video_caps(caps: &gst::Caps) -> Result<gst::Element, Error
                 gst::Element::link_many(&[&glupload, &glconvert, &glscale])?;
 
                 (glupload, glscale)
+            } else if feature.contains(NVMM_MEMORY_FEATURE) {
+                let queue = make_element("queue", None)?;
+                let nvconvert = make_element("nvvideoconvert", None)?;
+                nvconvert.set_property_from_str("compute-hw", "VIC");
```

Comment:
I'm not familiar with that element, can you document why you set those properties?

Reply:
@massis08 What hardware would be required to be able to run this? It sounds like a NVIDIA DeepStream element.
Is this generic across "all" NVIDIA APIs? 
On our NVIDIA Jetsons (embedded devices) the equivalent element is called `nvvidconv`.

Reply:
> I'm not familiar with that element, can you document why you set those properties?

**Compute-hw** is the type of computing hardware used for processing the conversion or scaling, it can be VIC for jetson or GPU for desktop x86. 
**Nvbuf-memory-type** is the type of memory to be allocated for the output buffers. In the case of being a jetson it has two options nvbuf-mem-default or nvbuf-mem-surface-array. In case of being x86 devices there are other options. 
To see more information about the gstreamer element and their properties: https://docs.nvidia.com/metropolis/deepstream/dev-guide/text/DS_plugin_gst-nvvideoconvert.html

> @massis08 What hardware would be required to be able to run this? It sounds like a NVIDIA DeepStream element. Is this generic across "all" NVIDIA APIs? On the Jetsons (embedded devices) using the `Accelerated GStreamer` the equivalent element is called `nvvidconv`: https://docs.nvidia.com/jetson/archives/r34.1/DeveloperGuide/text/SD/Multimedia/AcceleratedGstreamer.html#gstreamer-1-0-plugin-reference
> 
> I am not too familiar with DeepStream, but it appears as if there is overlap between DeepStream and the Accelerated GStreamer SDKs.

The nvvideoconvert is a DeepStream element and it was implemented for unifying interfaces of Jetson platforms and desktop GPUs. The nvvidconv can't be used in desktop GPUs.